src/main/java/com/school/project/controller/UserController.java

```diff
@@ -1,9 +1,51 @@
 package com.school.project.controller;
 
-import org.springframework.web.bind.annotation.RestController;
+import com.school.project.dto.UserDto;
+import com.school.project.repository.UserRepository;
+import com.school.project.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
 
 @RestController
+@RequestMapping
 public class UserController {
+    @Autowired
+    UserService userService;
+    @Autowired
+    UserRepository userRepository;
+
+    @PostMapping("/user/creation")
+    public void createUser(@RequestBody UserDto userDto) {
```

Comment:
Here is my first comment to all student projects:

Read this on REST API design principles and adjust end point names accordingly:
https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9

In short: we do not specify action like "create" in URL, but rather the HTTP method name (POST in this case) specifies the creational action. Therefore the URL should just look like "/user".